actionlist/app/actions/action_items.js

```diff
@@ -0,0 +1,6 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
+import { createAction } from 'redux-actions';
+import * as types from './types';
+
+export const setActionList = createAction(types.SET_ACTION_LIST);
```

Comment:
Is `setActionList` used anywhere? If not, it should be removed (here and in `types.js`).